### PR TITLE
Feat: Expose endpoint and created user in model serving list

### PR DESCRIPTION
**Pull request by gee05053**
This PR resolves issue https://github.com/lablup/backend.ai-webui/issues/2038

User can see endpoint URL and admin can see created user.

Admin screen
스크린샷 2023-11-23 오후 2 57 13

User screen
스크린샷 2023-11-23 오후 2 57 45

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: react/src/pages/ServingListPage.tsx [hunk 1]
```diff
@@ -146,6 +147,7 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
 
   const sortedEndpointList = _.sortBy(modelServiceList?.items, 'name');
 
+  console.log(baiClient);
```
Comment (rapsealk) [inline react/src/pages/ServingListPage.tsx]: Please remove any debugging logs prior to the code review.

Comment (gee05053) [reply react/src/pages/ServingListPage.tsx]: Sorry.. I missed that

File: react/src/pages/ServingListPage.tsx [hunk 2]
```diff
@@ -360,9 +359,7 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
                   ? {
                       title: t('modelService.CreatedUser'),
                       dataIndex: 'created_user',
-                      render: (created_user) => (
-                        <EndpointCreatedUserText userId={created_user} />
-                      ),
+                      render: (created_user) => created_user,
                     }
                   : {},
```
Comment (yomybaby) [inline react/src/pages/ServingListPage.tsx]: When `is_admin` is `false, `{}` is added to the array. It's not a good idea even `Table` doesn't show it. 
One of the solution is
```js
onst array = [
  1,
  2,
  ...(is_admin ? [3] : []),
  4
];
```

**Review by yomybaby — CHANGES_REQUESTED** [review]

File: react/src/pages/ServingListPage.tsx (hunk 2 above)
Comment (gee05053) [reply react/src/pages/ServingListPage.tsx]: Oh.. I didn't know that!😳

File: resources/i18n/en.json [hunk 3]
```diff
@@ -426,7 +426,8 @@
     "ServiceNameRule": "Only allow 4 to 24 characters of alphanumeric, underscore(_), hyphen(-), and period(.) and it must end with an alphanumeric character.",
     "FormValidationFailed": "Form validation failed",
     "resources": "Resources",
-    "Image": "Image"
+    "Image": "Image",
+    "CreatedUser": "Created User"
```
Comment (agatha197) [inline resources/i18n/en.json]:
```suggestion
    "Owner": "Owner"
```

**Review by agatha197 — APPROVED** [review]
LGTM

**Comment by gee05053** [comment]
This PR is closed because user config is wrong

**Pull request closed without merging**